backend/hardhat.config.ts

```diff
@@ -217,7 +217,7 @@ const config: HardhatUserConfig = {
       accounts,
     },
     'sapphire-localnet': {
-      url: 'http://localhost:8545',
+      url: 'http://127.0.0.1:8545',
```

Comment:
There should not be any difference.. What if localhost is 127.0.1.1?

Reply:
Not sure, I had weird problems with dockers communicating with each other.

Reply:
That would be worth investigating, because we're consistently using `http://localhost:8545` across the examples.

Interesting though is that `geth` registered the Chain ID `1337`: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-1337.json
and they're using http://127.0.0.1:8545 as well.

Reply:
Oh, perhaps ipv6 issues on your machine?